pages/linux/runuser.md

```diff
@@ -0,0 +1,24 @@
+# runuser
+
+> Run commands as a specific user and group IDs.
+> Do not ask for password. Only useful when run as root user.
+
+- Run 'command' under substitute 'user':
+
+`runuser {{user}} -c '{{command}}'`
+
+- Run 'command' under substitute 'user' and 'group':
+
+`runuser {{user}} -g {{group}} -c '{{command}}'`
+
+- Start a login shell as a specific user:
+
+`runuser {{user}} -l`
+
+- Specific a 'shell' for running or login instead of default shell:
```

Comment:
Perhaps this will read better?
```suggestion
- Start a specific shell instead of the default shell:
```

Reply:
`for running or login` is describing exactly the feature of `runuser`. Disagree.